cmd/main.go

```diff
@@ -425,6 +426,12 @@ func main() {
 		apiEnabler.Enable()
 	}()
 
+	go func() {
```

Comment:
it is not enough, we should add changes to yaml to add ports to service

Reply:
The goal is not to expose it to the public but keep it internal. Someone that needs/wants to use this would exec into the POD and run curl commands there.